package.json

```diff
@@ -108,7 +108,7 @@
     "oracledb": "^5.2.0",
     "pg": "^8.7.1",
     "pg-native": "^3.0.0",
-    "pino": "^5.17.0",
+    "pino": "^7.8.0",
```

Comment:
big jump, I assume you tested with 6 as well

Reply:
Yep. All good.